### PR TITLE
Since mysql5.6 version does not support timestamp default current_timestamp, modify sql creation statement（由于mysql5.6一下版本不支持timestamp default current_timestamp,修改sql创建语句）

### PR DESCRIPTION
**Pull request by c-f-cooper**

### CODE REVIEW

**Comment by boandai** [comment]
very good, @lenboo , please review ！

File: sql/create/release-1.0.0_schema/mysql/escheduler_ddl.sql [hunk 1]
```diff
@@ -176,8 +176,8 @@ CREATE TABLE `t_escheduler_project` (
   `desc` varchar(200) DEFAULT NULL COMMENT '项目描述',
   `user_id` int(11) DEFAULT NULL COMMENT '所属用户',
   `flag` tinyint(4) DEFAULT '1' COMMENT '是否可用  1 可用,0 不可用',
-  `create_time` datetime DEFAULT CURRENT_TIMESTAMP COMMENT '创建时间',
```
Comment (boandai) [inline sql/create/release-1.0.0_schema/mysql/escheduler_ddl.sql]: if `create_time` field doesn't have default value,  when create project, can save to database normally?  I'm not sure for this

if could add unit test will be better

File: docs/zh_CN/SUMMARY.md [hunk 2]
```diff
@@ -29,7 +29,7 @@
     * [开发环境搭建](后端开发文档.md#项目编译)
     * [自定义任务插件文档](任务插件开发.md#任务插件开发)
     
-* [接口文档](http://52.82.13.76:8888/easyscheduler/doc.html?language=zh_CN&lang=cn)
+* [接口文档](http://52.82.13.76:8888/escheduler/doc.html?language=zh_CN&lang=cn)
```
Comment (boandai) [inline docs/zh_CN/SUMMARY.md]: very good, thank you for your contribution!

Comment (c-f-cooper) [reply docs/zh_CN/SUMMARY.md]: 文档错误这里也没法合并吗

**Comment by c-f-cooper** [comment]
文档错误也没法合并么

**Comment by boandai** [comment]
Please wait a while. We are now planning to use PostgreSql as the default database.  maybe many changes will happen

**Comment by c-f-cooper** [comment]
> Please wait a while. We are now planning to use PostgreSql as the default database. maybe many changes will happen

ok

**Comment by qiaozhanwei** [comment]
Sorry, this is not only to remove the mysql CURRENT_TIMESTAMP, but also the default value of the program that needs to set the time, can not take the default value of the data.

不好意思，这个不单单是要把mysql的CURRENT_TIMESTAMP去掉，还有程序中的涉及到的需要设置时间的默认值，不能走数据的默认值了

**Comment by c-f-cooper** [comment]
ok

**Pull request closed without merging**